src/__tests__/behaviors/focusTrap.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import React from 'react'
+import {fireEvent, render} from '@testing-library/react'
+import {focusTrap} from '../../behaviors/focusTrap'
+
+async function nextTick() {
+  return new Promise(resolve => {
+    process.nextTick(resolve)
+  })
+}
+
+it('Should initially focus the first focusable element when activated', () => {
+  const {container} = render(
+    <div>
+      <button tabIndex={0}>Bad Apple</button>
```

Comment:
Curious why you needed to put `tabIndex={0}` on these, shouldn't they already by tubbable since they are buttons?

Reply:
Because Jest/JSDOM 😂

Reply:
Actually, I might be able to remove them. I just figured since it's unrelated to what I'm actually testing, I didn't want to leave it to chance. I think I saw something on Stackoverflow where someone said that they had to include `tabindex="0"` on everything they wanted to call `.focus()` on.

Reply:
Ah, nope, confirmed that Jest won't let you `.focus()` any element that doesn't explicitly have tabindex set to a non-negative value.

Reply:
Ok sounds good!